docs/configuration-options.md

```diff
@@ -12,110 +12,65 @@ function configureSphinx() public override {
 - [DevOps Platform Options](#devops-platform-options)
-  - [`string orgId`](#string-orgid)
-  - [`Network[] mainnets`](#network-mainnets)
-  - [`Network[] testnets`](#network-testnets)
+  - [`string orgId`](#string-orgid-required)
+  - [`string[] mainnets`](#string-mainnets-optional)
+  - [`string[] testnets`](#string-testnets-optional)
 
 ## Configuration Options
 
-### `address[] owners`
+### `address[] owners` (Required)
 ```
 sphinxConfig.owners = [0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266];
 ```
 
 The list of addresses that own your Gnosis Safe. Owners must approve transactions before they can be executed.
 
-### `uint256 threshold`
+### `uint256 threshold` (Required)
 ```
 sphinxConfig.threshold = 1;
 ```
 
 The number of owners required to approve transactions.
 
-### `string projectName`
+### `string projectName` (Required)
 
 ```
 sphinxConfig.projectName = "My_Project";
 ```
 
 The name of your project, which is the top-level directory for your [deployment artifacts](https://github.com/sphinx-labs/sphinx/blob/main/docs/deployment-artifacts.md).
 
-### `uint256 saltNonce`
+### `uint256 saltNonce` (Required)
 
 An optional nonce which is one of the inputs that determines the `CREATE2` address of a Gnosis Safe. Changing this to a new value will cause a Gnosis Safe to be deployed at a new address. Defaults to `0`.
 
 ## DevOps Platform Options
 
-### `string orgId`
+### `string orgId` (Required)
 
 ```
 sphinxConfig.orgId = "abcd-1234";
 ```
 
 Your organization ID from the Sphinx UI (under "Options" -> "API Credentials"). This is a public field, so you don't need to keep it secret.
 
-### `Network[] mainnets`
+### `string[] mainnets` (Optional)
 
 ```
-sphinxConfig.mainnets = [Network.ethereum, Network.arbitrum];
+sphinxConfig.mainnets = ["ethereum", "optimism", "arbitrum"];
 ```
 
-The list of production networks to deploy on.
-
-Valid values:
-
-| Network | `Network` enum |
-| ----------- | ----------- |
-| Ethereum | `Network.ethereum` |
-| Optimism | `Network.optimism` |
-| Arbitrum | `Network.arbitrum` |
-| Polygon PoS | `Network.polygon` |
-| Binance Smart Chain | `Network.bnb` |
-| Gnosis Chain | `Network.gnosis` |
-| Linea | `Network.linea` |
-| Polygon zkEVM | `Network.polygon_zkevm` |
-| Avalanche C-Chain | `Network.avalanche` |
-| Fantom | `Network.fantom` |
-| Base | `Network.base` |
-| Scroll | `Network.scroll` |
-| Celo | `Network.celo` |
-| Moonbeam | `Network.moonbeam` |
-| Moonriver | `Network.moonriver` |
-| Fuse | `Network.fuse` |
-| Evmos | `Network.evmos` |
-| Kava | `Network.kava` |
-| OKT Chain | `Network.oktc` |
-| Rootstock | `Network.rootstock` |
-
-### `Network[] testnets`
+A list of production networks to propose on via the `--networks mainnets` flag. Provides a convenient way to propose on many networks without needing to specify them on the command line. Note that the strings in this array must match the network names in the `rpc_endpoints` section of your `foundry.toml` file.
```

Comment:
It’s probably worth noting somewhere that when using the propose command, you are limited to the networks we support and then maybe just link to the table in the main readme.